### PR TITLE
Fledging content

### PR DESCRIPTION
**Pull request by ateucher**
Content taken and expanded from the [AGU fledging poster](https://docs.google.com/presentation/d/1p5qVbSqUij9FQ9yf7deSVVuXGM6GYjcAyRA78ZtsSxA/edit#slide=id.g31ac2630191_1_0), and conversations around that.

Deploy preview: https://deploy-preview-15--openscapes-cloud-preview.netlify.app/fledging

### CODE REVIEW

File: fledging.qmd [hunk 1]
```diff
@@ -6,7 +6,76 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
...
+When thinking about cloud computing, we consider two components: where is our 
+data? And where is our compute? In general, if our data is already on our local
+computer, we will work with that data on our computer, so the scope we are 
+considering is how to work with *data that is hosted in the cloud*.
 
-More resources coming soon...
+![](images/data-compute-cloud-grid.png){width="400" fig-align="center"}
+
+It is also important to note the environmental costs of cloud computing - 
+it is [carbon-intensive](https://mit-serc.pubpub.org/pub/the-cloud-is-material/release/2), and so it is worth taking this into consideration when
```
Comment (ateucher) [inline fledging.qmd:17]: This is an important but complex topic, I don't want to overreach...

**Comment by ateucher** [comment]
Note: Should add this link as a resource: https://developmentseed.org/scaling_science

File: fledging.qmd (hunk 1 above)
Comment (ateucher) [reply fledging.qmd:17]: See discussion [here](https://openscapes.slack.com/archives/CHZ2SMN7J/p1733966801974699)

File: fledging.qmd [hunk 2]
```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
 and want to continue their adventures in cloud computing, have the knowledge
 and resources to develop their own solutions.
```
Comment (cboettig) [inline fledging.qmd:7]: I think we might want a little intro contextualizing how we see and experience the challenge of fledging? As discussed in the slack, this is somewhat the inverse of how 'cloud' is often presented (where new users begin in laptop-based installations first and must then learn how (and when) to migrate those workflows to cloud.  The idea of users who can run analyses on the cloud while actually not knowing how or even if they could run it on their own laptop can frankly be mind-blowing to a certain slice of even quite sophisticated computational users.

File: fledging.qmd [hunk 3]
```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
...
+
+### What are fledging paths for **compute**?
+
+Depending on computing needs and data size, there are many options for fledging.
+These may include:
+
+- Continue compute in the cloud using a shared environment (example: JupyterHub)
+    - If your institution offers a JupyterHub instance, this is a great option 
+      to seamlessly transfer your learnings from the Openscapes Hub to a Hub
+      for "real science".
```
Comment (cboettig) [inline fledging.qmd:45]: At some point we probably need to raise the issue of dependency management... e.g. I see students work in a hosted jupyter env, and might assume all jupyter envs have the same set of packages available ...

File: fledging.qmd [hunk 4]
```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
...
+      cloud data access patterns (such as streaming and range requests) without 
+      worrying about cloud costs and administration.
+- Compute locally using a shared environment “container” (example: Docker, 
+  Codespaces)
+    - One of the great benefits that the shared JupyterHub offers is a computing
+      environment that is already set up, so you don't have to go through the pain
+      of installing and configuring everything yourself. The images used in the 
+      JupyterHubs can often be used locally using tools like Docker. See some 
+      examples [here](https://pangeo-docker-images.readthedocs.io/en/latest/howto/launch.html#how-to-launch-jupyterlab-locally-with-one-of-these-images).
+
```
Comment (cboettig) [inline fledging.qmd:65]: I know jupyterlite doesn't support all of our packages yet, but I think it's an intriguing option!

File: fledging.qmd [hunk 5]
```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
...
+- Compute locally with cloud-native workflows (example: streaming cloud data 
+  with earthaccess + xarray)
+    - For data and analysis jobs that can fit within your laptop's resrouces 
+      (RAM and hard disk), working locally with the same tools and workflows 
+      learned from working in the hub is a great option, and computing in the 
+      cloud is likely unnecessary. You can benefit greatly from the effiency of 
+      cloud data access patterns (such as streaming and range requests) without 
+      worrying about cloud costs and administration.
+- Compute locally using a shared environment “container” (example: Docker, 
+  Codespaces)
```
Comment (cboettig) [inline fledging.qmd:59]: Seems odd to include codespaces under 'compute locally'?

**Review by cboettig — COMMENTED** [review]
a few thoughts but looks awesome!

**Comment by ateucher** [comment]
Thanks for the review @cboettig. Great points, I'll wait for @jules32 to have a look and then do another pass.

File: fledging.qmd (hunk 2 above)
Comment (jules32) [inline fledging.qmd:7]: The idea of "fledging" and how to support users is a work-in-progress developed with the NASA Openscapes Mentor community, and this webpage will be periodically updated to reflect current thinking and experiences.

File: fledging.qmd (hunk 4 above)
Comment (ateucher) [reply fledging.qmd:65]: Definitely! I haven't followed its development closely, do you think it's worth mentioning here?

File: fledging.qmd (hunk 2 above)
Comment (ateucher) [reply fledging.qmd:7]: Thanks @jules32 - sorry I didn't see where your suggested text is?

**Comment by ateucher** [comment]
Thanks @jules32 and @cboettig! I've made some edits to address your comments - let me know if you think it needs more.

**Pull request merged**